homeWork7-8/getTriangleP.cpp

```diff
@@ -0,0 +1,18 @@
+#include <iostream>
+#include <cmath>
+#include <iomanip>
+
+using namespace std;
+
+double SegmentLength(const double& x1, const double& y1, const double& x2, const double& y2) {
```

Comment:
built-in type-երի համար պետք չի գրել const &, իրանք առանց դրա ավելի արագ են

Reply:
dzec :D